mantidproject/mantid
# Doctests for training courses

This adds the capability to doctest wiki pages and uses the python introduction as an example

Further work is required to integrate this into the builds

How to add doctests to mwiki pages is documented here
http://www.mantidproject.org/Category:Tested_Examples

And this is the category that any pages with doctests should be added.

fixes #13310 
### To test
1. Run the script
   python.exe TestWikiPython.py -o C:\Mantid\Code\Mantid\docs\source\mwTests
2. check it produces a collection of rst, doctest files
3. run the doctest build (if the output directory was under \docs\source they will be included)
4. The new tests should all pass
